README.md

```diff
@@ -1 +1,171 @@
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live demo](#🚀live-demo)
+    - [Presentation](#presentation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Budget App] <a name="about-project"></a>
+
+💰💼 Budget App is a web-based budgeting application built using Ruby on Rails and PostgreSQL.
+With BudgetApp, users can effortlessly track their income and expenses, gaining full control over their finances.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Gitflow execution
+- Register an account
+- Login registered account
+- Create categories and transactions
+- Upload Profile photo
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 🚀Live Demo <a name="Live Demo"></a>
+
+[Live](https://microo.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Presentation <a name="Presentation"></a>
+
+[presentation](https://www.loom.com/share/55f60a409b4f4fcc95e684ce38c93f56?sid=fa02a167-aa1e-4d21-906f-70208065ccf4)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Setup
+
+- Clone this repository to your desired folder:
+
+for cloning follow this steps
+
+- Create a local directory where you can clone the project
+- Clone the project to your directory by running
+  https://github.com/jeanbuhendwa/ror-budget-app.git
+- Run the code with ruby
+
+Example commands:
+
+```sh
+  git clone https://github.com/jeanbuhendwa/ror-budget-app.git
+```
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Node.js
+- npm
+- ruby 2.7.0
+- rails
+- PostgreSQL
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Install this project with:
+
+cd 'project name'
+
+"npm i" to install all dependencies
+
+"bundle install" to install all gems
+
+"rails s" to run the server
+
+### Run tests
+
+- "rspec" to run the tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Jean-Jacques Buhendwa**
+
+- GitHub: [jeanbuhendwa](https://github.com/jeanbuhendwa)
+
```

Comment:
- [OPTIONAL] I think it would be better to add more contact info like `LinkedIn, Twitter` so people can have a chance to connect with you via other social media 😉